{{cookiecutter.project_slug}}/Makefile

```diff
@@ -122,8 +126,56 @@ clean-all: clean-code clean-data clean-notebooks  ### Delete code, data, and not
 
 ##### JOBS #####
 
-.PHONY: train
-train:  ### Run a training job
+.PHONY: develop
+develop: upload-code upload-notebooks  ### Run a development job
+	$(NEURO) run \
+		--name $(DEVELOP_JOB) \
+		--preset $(PRESET) \
+		--detach \
+		--volume $(DATA_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(DATA_DIR):ro \
+		--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):rw \
+		--volume $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
+		--env EXPOSE_SSH=yes \
+		--env JOB_LIFETIME=1d \
+		$(CUSTOM_ENV_NAME) \
+		"sleep infinity"
+
+.PHONY: connect-develop
+connect-develop:  ### Connect to the remote shell running on the development job
+	$(NEURO) exec --no-key-check $(DEVELOP_JOB) bash
+
+.PHONY: logs-develop
+logs-develop:  ### Connect to the remote shell running on the development job
+	$(NEURO) logs $(DEVELOP_JOB)
+
+    .PHONY: remote-debug-develop
+remote-debug-develop:  ### Forward SSH port to localhost for using PyCharm Pro remote debugging
+	@test ${REMOTE_DEBUG_PORT} || { echo 'Please set up env var REMOTE_DEBUG_PORT'; false; }
+	@echo
+	@echo 'Enabling remote debugging with PyCharm Pro:'
```

Comment:
Maybe use vscode? At least it's free.

Reply:
OK let's move this to documentation

Reply:
if the setup is the same there, it's easy. Instructions will be on neu.ro docs